tests/zfs-tests/tests/functional/raidz/raidz_002_pos.ksh

```diff
@@ -36,6 +36,6 @@
 #	is set to 5min, but actual runtime might be longer.
 #
 
-log_must $RAIDZ_TEST -S -t 300
+log_must $RAIDZ_TEST -S -t 40
```

Comment:
In exactly what environment does this need to be reduced.  Recent automated test runs from master show that this test does take roughly 300 seconds to run.  

```
Test: /usr/share/zfs/zfs-tests/tests/functional/raidz/raidz_002_pos (run as root) [05:04] [PASS]
```